task.sql

```diff
@@ -1,2 +1,5 @@
 # Write your SQL code for the database creation here. Good luck! 
 USE ShopDB; 
+
+CREATE INDEX Name ON Products (Name);
+CREATE INDEX Email ON Customers (Email);
```

Comment:
![image](https://github.com/user-attachments/assets/87656d15-dac3-4808-ad77-5a4260729102)
Add Eof(End-of-file), need to add blank line at the end